config.yaml

```diff
@@ -150,6 +150,10 @@ evidence:
     output_filename: eva.json.gz
     output_spark_dir: eva
     path: evidence-files
+  - bucket: otar012-eva/pharmacogenomics
+    output_filename: pharmacogenomics.json.gz
+    output_spark_dir: pharmacogenomics
+    path: evidence-files
```

Comment:
PGx is technically a different data type than the evidence files. I'd consider this data as a separate data type, similarly as we treat mouse phenotypes data.